python/lsst/summit/utils/utils.py

```diff
@@ -0,0 +1,392 @@
+# This file is part of summit_utils.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+from scipy.ndimage.filters import gaussian_filter
+import lsst.afw.detection as afwDetect
+import lsst.afw.math as afwMath
+import lsst.pipe.base as pipeBase
+import lsst.utils.packages as packageUtils
+
+from lsst.obs.lsst.translators.lsst import FILTER_DELIMITER
+from lsst.obs.lsst.translators.latiss import AUXTEL_LOCATION
+
+from astro_metadata_translator import ObservationInfo
```

Comment:
Is this the correct namespace for `astro_metadata_translator`?

Reply:
Yup, it's actually standalone and lives outside the stack! Give it a go 🙂